src/classes/groundplaces.ts

```diff
@@ -103,7 +104,7 @@ export class GroundPlacesController {
    * @returns {void}
    */
   public updateStopCluster(stopClusterGpuid: StopClusterGpuid, propertiesToUpdate: UpdateStopProperties): void {
-    const copyGroundPlaces: GroundPlaces = this.storageService.getGroundPlaces();
+    const copyGroundPlaces: GroundPlaces = cloneObject(this.storageService.getGroundPlaces());
```

Comment:
Why use a method like that ?
and not keep the logic inside the storage ? 

this.storageService.getGroundPlacesCopy() and type it well.